src/odemis/driver/test/hamamatsurx_test.py

```diff
@@ -19,13 +19,15 @@
 from __future__ import division
 
 import logging
+import os
+
 from odemis import model, util
 from odemis.driver import hamamatsurx
 import time
 from odemis.driver import andorshrk
 
 import unittest
-from unittest.case import skip
+from unittest.case import skip  # TODO needed??
```

Comment:
Only needed if you want to add a `@skip` in front of a test case. You can drop it.